blazor/skeleton/getting-started.md

```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 This section briefly explains about how to include Blazor Skeleton component in your Blazor Server App and Blazor WebAssembly App using Visual Studio.
 
+To get start quickly with Skeleton Component using Blazor, you can check on this video:
+
+{% youtube
+"youtube:https://www.youtube.com/watch?v=Zd6gzsjunYM&list=PLDzXQPWT8wEDGN8evydx9L-KpuW1ZxDy1&index=4"%}
```

Comment:
Change the URL to point only to the corresponding video and not the playlist

Also include the Video URL's for FAB and SpeedDial as well

SpeedDial: https://www.youtube.com/watch?v=geUFOzkTFyY
FAB: https://www.youtube.com/watch?v=own8zVAakTk
Skeleton: https://www.youtube.com/watch?v=Zd6gzsjunYM

Reply:
I have updated the mentioned changes.